docker/.env

```
@@ -239,6 +239,11 @@ SMQ_USERS_HTTP_HOST=users
 SMQ_USERS_HTTP_PORT=9002
 SMQ_USERS_HTTP_SERVER_CERT=
 SMQ_USERS_HTTP_SERVER_KEY=
+SMQ_USERS_GRPC_HOST=users
+SMQ_USERS_GRPC_PORT=7002
+SMQ_USERS_GRPC_SERVER_CERT=${GRPC_MTLS:+./ssl/certs/domains-grpc-server.crt}${GRPC_TLS:+./ssl/certs/domains-grpc-server.crt}
```

dotenv-linter (4.0.0)

[warning] [SubstitutionKey] The SMQ_USERS_GRPC_SERVER_CERT key is not assigned properly

(SubstitutionKey)